schema-statistics/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Tracking Postgres schema statistics in pganalyze'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+As part of the "Schema Statistics" functionality, pganalyze allows tracking workload patterns, schema and data model design, as well as changes over time.
+
+## Breakdown of table and index sizes
+
+![](table-treemap.png)
+
+In the schema statistics overview you can find two treemaps, as well as full lists of all the tables and indexes on a given database. The treemap is useful to identify how a specific index or table size relates to the total.
```

Comment:
Definitely nitpick but I got confused about this "two treemaps", because I only see one treemap _in Tables tab_. I think when you say "overview", you meant all tabs over there, but since there is no tab called "overview", I thought you were talking about the Tables tab.
I think it's okay to leave it as is, but if there is any better way to rewrite this, that'd be great. Uhmm maybe something like "In the schema statistics Tables and Indexes tabs,"?